config/environments/development.rb

```diff
@@ -108,5 +108,7 @@
                   projects: '',
                   work: '' }
 
+  config.ss_data_release_url = 'http://localhost:3000'
```

Comment:
Please could you change this to config.data_release_url (so it's consistent with the Projects app)